### PR TITLE
New routing for url for section and part search

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves # EREGCSC-1333, EREGCSC-1324

**Description-**

**This pull request changes...**

- expected change 1
The query string for searching for supplemental content.  New version prevents selected sections and subparts are tied to a part.

Also modifies chip to a new format display part and subparts correctly.

**Steps to manually verify this change...**

1. steps to view and verify change

V1. Click on the resources page on prototypes, select a couple filters, will update the query string and results. 

V2.  Select a part on from the prototype.  Click a view resources box.  Click view all resources.

### CODE REVIEW

**Comment by PhilR8** [comment]
This PR isn't quite ready for review yet - it still needs to format the chip labels per this slack convo:

https://fearless.slack.com/archives/C02DE0DU6KU/p1651849129614469

**Comment by thwalker6** [comment]
> ###

It was.  Chip formatting was different story.  Added the logic in to format the chips correctly though so it is now included.

File: solution/ui/prototype/src/views/Resources.vue [hunk 1]
```diff
@@ -261,24 +260,53 @@ export default {
...
             return returnArr;
         },
+
         async getSupplementalContent(dataQueryParams, searchQuery) {
             this.isLoading = true;
             if (dataQueryParams?.part) {
                 const queryParamsObj = { ...dataQueryParams };
+                const parts = queryParamsObj.part.split(",");
                 queryParamsObj.part = queryParamsObj.part.split(",");
+                let partDict = {};
```
[the remark this answers is not in the document]
Comment (thwalker6) [reply solution/ui/prototype/src/views/Resources.vue:270]: Changed most of the let to const.

**Review by knollfear — APPROVED** [review]
lgtm  Thanks.

**Pull request merged**